Makefile

```
@@ -771,6 +767,145 @@ otel-forward: ## Port-forward Grafana (3000)
+		-e USERS=$(PERF_USERS) \
+		-e DURATION=$(PERF_DURATION) \
+		tests/perf/k6/concurrency-levels.js 2>&1 | tee $(PERF_OUT_DIR)/k6-steady-$(PERF_USERS)vu.log
+	@go run ./tests/perf/cmd/report \
+		-csv $(PERF_OUT_DIR)/k6-steady-$(PERF_USERS)vu.csv \
+		-title "MCP Gateway Steady State: $(PERF_USERS) VUs for $(PERF_DURATION)" \
+		-out $(PERF_OUT_DIR)/mcp-report.html
+	@echo "Results saved to $(PERF_OUT_DIR)/"
+	@echo "  open $(PERF_OUT_DIR)/mcp-report.html"
+
+.PHONY: perf-run-ramp
+perf-run-ramp: perf-build-k6 ## Run ramp-up test with profiling
```

checkmake (0.3.2)

[warning] Target body for "perf-run-ramp" exceeds allowed length of 5 lines (54).

(maxbodylength)